src/google-classroom-extractor/edfi_google_classroom_extractor/config.py

```diff
@@ -39,14 +38,14 @@ def get_sync_db_engine() -> sqlalchemy.engine.base.Engine:
     """
     running_in_notebook: bool = _is_running_in_notebook()
     logger.debug("Running in Jupyter Notebook: %s", running_in_notebook)
-    sync_database_directory = (
-        os.path.join("..", SYNC_DATABASE_LOCATION_SUFFIX)
+    sync_database_path = (
+        os.path.join("..", sync_database_directory)
```

Comment:
Again, wondering why trying to make this relative to the parent directory. Also what happens if `sync_database_directory` is a full path already, not a relative path?

Reply:
It only happens when is running in a notebook, I don't remember why this validation was added. 🤔